react/src/components/card/SprkCard.stories.js

```diff
@@ -11,18 +15,29 @@ export default {
   ],
   component: SprkCard,
   parameters: {
-    subcomponents: { SprkCardTeaser, SprkCardHighlightedHeader },
+    subcomponents: {
+      SprkHeading,
+      SprkText,
+      SprkStack,
+      SprkLink,
+      SprkStackItem,
+      SprkIcon,
+    },
     jest: ['SprkCard'],
     info: `${markdownDocumentationLinkBuilder('card')}`,
   },
 };
 
 export const defaultStory = () => (
-  <SprkCard
-    idString="card-1"
-    additionalContentClasses="sprk-o-Stack sprk-o-Stack--large"
-  >
-    Base Card Content
+  <SprkCard idString="default">
+    <SprkStack
+      additionalClasses="sprk-o-Stack__item sprk-c-Card__content"
```

Comment:
Related to [this comment]( https://github.com/sparkdesignsystem/spark-design-system/pull/3912/files#r591824056)

SprkCard contains the parent container that relates to Stack__item. 

My instinct here is to make this less opinionated and to instead move the SprkStack so then the user can easily understand where the `sprk-o-Stack__item` is related to.

```
<SprkCard>
     <SprkStack
          <SprkStack additionalClasses="sprk-o-Stack__item sprk-c-Card__content"
               <SprkStackItem
               <SprkStackItem
</SprkCard>
```

Reply:
responded up above, resolving this one